simulation/simple_sensor_simulator/src/vehicle_simulation/ego_entity_simulation.cpp

```diff
@@ -365,6 +376,29 @@ auto EgoEntitySimulation::setStatus(const traffic_simulator_msgs::msg::EntitySta
   setAutowareStatus();
 }
 
+auto EgoEntitySimulation::getBehaviorParameter() const
+  -> traffic_simulator_msgs::msg::BehaviorParameter
+{
+  return behavior_parameter_;
+}
+
+auto EgoEntitySimulation::setBehaviorParameter(
+  const traffic_simulator_msgs::msg::BehaviorParameter & behavior_parameter) -> void
+{
+  behavior_parameter_ = behavior_parameter;
+}
```

Comment:
Class `EgoEntitySimulation` has almost only [private fields](https://github.com/tier4/scenario_simulator_v2/blob/477d2deffe4eb89ea60f96d89879c6cf9f3b60d7/simulation/simple_sensor_simulator/include/simple_sensor_simulator/vehicle_simulation/ego_entity_simulation.hpp#L44-L69). It also has multiple getters and setters. In my opinion it is worth keeping it consistent and leaving `behavior_parameter_` like this even though this setter does not perform additional operations. 

Personally, when I operate on a class I prefer to set the values via set** (useful with Intelligent Code Completion) rather than thinking which fields of the class are public and I can assign them directly and which have setters. The code also seems more readable to me if outside the class only its setters are used and not a mixed setter with assignments.

However, this is my personal feeling, please let me know if you agree, if not I will of course make `behavior_parameter_` public and remove the getter and setter.

Reply:
When adding or changing elements to the code, it is good practice to follow the style of the original code. Therefore, your opinion is more correct than mine.

I withdraw this change request.